src/core/server/plugins/plugins_system.ts

```diff
@@ -66,15 +66,14 @@ export class PluginsSystem {
       return contracts;
     }
 
-    const sortedPlugins = this.getTopologicallySortedPluginNames();
-    this.log.info(`Setting up [${this.plugins.size}] plugins: [${[...sortedPlugins]}]`);
-
-    for (const pluginName of sortedPlugins) {
-      const plugin = this.plugins.get(pluginName)!;
-      if (!plugin.includesServerPlugin) {
-        continue;
-      }
+    const sortedPlugins = new Map(
+      [...this.getTopologicallySortedPluginNames()]
+        .map((pluginName) => [pluginName, this.plugins.get(pluginName)!] as [string, PluginWrapper])
+        .filter(([pluginName, plugin]) => plugin.includesServerPlugin)
+    );
+    this.log.info(`Setting up [${sortedPlugins.size}] plugins: [${[...sortedPlugins.keys()]}]`);
```

Comment:
Not really the scope of the PR, but maybe this should be `${[...sortedPlugins].join(', ')}` instead to improve readability?